cli/main.go

```diff
@@ -112,13 +115,25 @@ func Main() {
 		log.Fatal("Please specify a relay timeout greater than 0")
 	}
 
+
+	if *relayMinBidEth < 0.0 {
+		log.Fatal("Please specify a non-negative minimum bid")
+	}
+
+	if *relayMinBidEth > 100.0 {
```

Comment:
I don't see any reason to hardcode an arbitrary max minBid.  I think it should be up to the user to define any valid positive number.  Additionally mev boosts could be run a future forks where a high absolute number could be normal.